server/log/router.ts

```diff
@@ -14,8 +21,51 @@ export const LogModule: UIServerModule = {
     const routerForModule = express.Router();
 
     // implementation to follow
-    routerForModule.get('*', authFn, (req, res) => res.sendStatus(200));
+    routerForModule.get('*', authFn, (req, res) => {
+      const { isWs } = req as strimziUIRequestType;
+      const { ws } = res as strimziUIResponseType;
+      if (isWs) {
+        ws.on('message', handleMessage(logger));
+        ws.on('close', handleClose(logger));
+      } else {
+        // Return 426 Upgrade Required if this isn't a websocket request
+        res.sendStatus(426);
+      }
+    });
 
     return exit({ mountPoint: '/log', routerForModule });
   },
 };
+
+const handleMessage: (logger: Logger) => (data: Data) => void = (logger) => (
```

Comment:
Pedantic, but can you rename to `messageHandler` (as it's not itself handling it, but the return function.

Reply:
Done